makefiles/info-global.inc.mk

```diff
@@ -9,6 +9,8 @@ FEATURES_CONFLICT_GLOBAL := $(FEATURES_CONFLICT)
 FEATURES_CONFLICT_MSG_GLOBAL := $(FEATURES_MSG_CONFLICT)
 DISABLE_MODULE_GLOBAL := $(DISABLE_MODULE)
 DEFAULT_MODULE_GLOBAL := $(DEFAULT_MODULE)
+FEATURES_BLACKLIST_GLOBAL := $(FEATURES_BLACKLIST)
+DISABLE_MODULE_GLOBAL := $(DISABLE_MODULE_GLOBAL)
```

Comment:
This line seems invalid and DISABLE_MODULE_GLOBAL is already set 3 lines above.

Reply:
removed